webdriver/tests/bidi/script/add_preload_script/invalid.py

```diff
@@ -204,6 +204,14 @@ async def test_params_contexts_empty_list(bidi_session):
         ),
 
 
+async def test_params_contexts_invalid_value(bidi_session):
```

Comment:
We actually have already a test for this for preload script: https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/bidi/script/add_preload_script/invalid.py#L216-L222